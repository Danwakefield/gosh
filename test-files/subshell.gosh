echo "4 test cases"
echo $(echo "SUCCESS 1")
echo $(echo $(echo "SUCCESS 2"))
echo $(
if true; then
	echo "SUCCESS 3"
fi
)
A="SUCCESS 4"
true $( A="FAIL 4" )
echo $A
