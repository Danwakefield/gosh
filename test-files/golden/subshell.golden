4 test cases
SUCCESS 1
SUCCESS 2
SUCCESS 3
SUCCESS 4
